Integrations/integration-Netskope.yml

```diff
@@ -225,13 +225,45 @@ script:
         return {
             Type: entryTypes.note,
             ContentsFormat: formats.json,
-            Contents: retArray,
+            Contents: result.data,
             ReadableContensFormat: formats.markdown,
             HumanReadable: tableToMarkdown('Netskope Events', retArray, headers),
             EntryContext: ec
         };
     }
 
+    function now() {
+        return parseInt(new Date().getTime()/1000)
+    }
+
+    function fetchAlerts() {
+        last_run = parseInt(getLastRun())
+        if (!last_run) {
+            last_run = now() - parseInt(translateTimeperiod('Last24Hours'))
+        }
+
+        args = {}
+        args.type = ""
+        args.starttime = last_run.toString()
+        args.endtime = now().toString()
+
+        alerts = getAlerts().Contents
+
+        incidents = []
+        alerts.forEach(function(item) {
+            incident = {}
+            d = new Date(0)
```

Comment:
cant we just do here `var d = new Date(item.timestamp).toISOString();` ?

Reply:
fixed